test/test.go

```diff
@@ -505,6 +505,10 @@ func (h *Helper) Path(name string) string {
 		joined = filepath.Join(h.tempdir, name)
 	}
 
+	if _, err := os.Stat(joined); os.IsNotExist(err) {
+		return ""
+	}
```

Comment:
Yeah, so we can take out `lock.json` and poss (as above) the `manifest.json` if not needed in the `final` folder.
Reason I put it here (not that it feels right) is that we don't get into `CompareFile` as symlink  check below fails. 

We take another look in the morning at how I might be able to move this but recommendations welcome.